.github/workflows/android.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        type: [manager, libs, apps, manager-vcpkg, libs-vcpkg, apps-vcpkg, libs-cmake]
+        type: [manager-vcpkg, libs-vcpkg, apps-vcpkg, libs-cmake]
```

Comment:
Since there are no more native builds, there is no reason to keep '-vcpkg' suffix.
Please remove